server/lib/db/index.js

```diff
@@ -200,6 +200,14 @@ class DatabaseConnector {
       if (data[key] && typeof data[key] === 'string') {
         data[key] = this.bid(data[key]);
       }
+
+      // the key exists on the object and value is an array
+      if(data[key] && Array.isArray(data[key])) { 
```

Comment:
This is a cool improvement. :+1:

At some point, we should start writing unit tests for these server-side functions.